assets/ocp-performance/variables.libsonnet

```diff
@@ -3,8 +3,7 @@ local var = g.dashboard.variable;
 
 {
   datasource:
-    var.datasource.new('datasource', 'prometheus')
-    + var.datasource.withRegex('/^Cluster Prometheus$/'),
```

Comment:
Using a dashboard against a prometheus outside of openshift, I update this variable after importing. Instead, just deleting for good.

Reply:
Confirmed on ROSA cluster that the dashboard variable auto-populates to 'Cluster Prometheus'